src/libs/ReportUtils.js

```diff
@@ -411,18 +412,74 @@ function formatReportLastMessageText(lastMessageText) {
 }
 
 /**
- * Helper method to return a default avatar
- *
+ * Helper method to return the default avatar associated with the given login
  * @param {String} [login]
  * @returns {String}
  */
 function getDefaultAvatar(login = '') {
-    // There are 8 possible default avatars, so we choose which one this user has based
+    // There are 24 possible default avatars, so we choose which one this user has based
+    // on a simple hash of their login
+    if (!login) {
+        return Expensicons.FallbackAvatar;
+    }
+    if (login === CONST.EMAIL.CONCIERGE) {
+        return Expensicons.ConciergeAvatar;
+    }
+
+    const loginHashBucket = (Math.abs(hashCode(login.toLowerCase())) % 24) + 1;
```

Comment:
Will it be an overkill if we want to move `24` to CONST and also `Math.abs` line to a utils method, as we also also use that in `getOldDorDefaultAvatar`.